File: src/main/resources/scripts/engine.pl
use JSON;
use org.folio.rest.camunda.utility.ScriptEngineUtility;

sub %s($) {
  my ($inArgs) = @_;
  my $scriptEngineUtility = ScriptEngineUtility();
  my $args = scriptEngineUtility.decodeJson($inArgs);
  my $returnObj = scriptEngineUtility.createJson();
  %s
  return scriptEngineUtility.encodeJson($returnObj);
}

File: src/main/resources/scripts/engine.py
import json
import org.folio.rest.camunda.utility.ScriptEngineUtility;

def %s(inArgs):
  scriptEngineUtility = org.folio.rest.utility.ScriptEngineUtility();
  args = scriptEngineUtility.decodeJson(inArgs);
  returnObj = scriptEngineUtility.createJson();
  %s
  return scriptEngineUtility.encodeJson(returnObj);
